src/proposer/proposer_logic.cpp

```diff
@@ -44,32 +44,37 @@ class LogicImpl final : public Logic {
   boost::optional<proposer::EligibleCoin> TryPropose(const staking::CoinSet &eligible_coins) override {
     AssertLockHeld(m_active_chain->GetLock());
 
-    const CBlockIndex *current_tip =
-        m_active_chain->GetTip();
+    const CBlockIndex *current_tip = m_active_chain->GetTip();
     if (!current_tip) {
       return boost::none;
     }
-    const blockchain::Height current_height =
-        m_active_chain->GetHeight();
-    const blockchain::Height target_height =
-        current_height + 1;
+    const blockchain::Height current_height = m_active_chain->GetHeight();
+    const blockchain::Height target_height = current_height + 1;
```

Comment:
These three changes seem unnecessary.

Reply:
There is no need to have less readable declarations on multiple lines while they don't even break the 80 chars rule.

Reply:
It's true, my point is that these changes are not relevant to the PR at all. They should go to a separate code-style PR.

Imagine, I use `git blame` on this file and see your changes. Then I find your commit "new rpc whatever", and will need to realize what is the impact of changed lines to new RPC commands.

Reply:
If you have to check what happened a history comparison will immediately show that the line was just reindented, while I agree to keep refactoring and renaming separate from feature PRs this does not mean that we should refrain from cleaning stuff around if the changes are limited.

Reply:
I changed my mind, I will factor those changes out and maybe create a PR later on those.